### ISSUE TITLE
Modelling different elasto-plastic behaviour in tension and compression

### ISSUE THREAD

**Issue by alda30** [mkraska/CalculiX-Examples#52]
Hi,

I'm trying to model a fibre reinforced concrete beam with different stiffness and stress-strain curve in tension and compression.
How should I implement this model?
Can I simply add the compression with negative values as below: (example numbers)
*plastic
-1200,-1
-100,-0.1
20,0.1
120,1

Thanks in advance for your help and assistance.

Alex

**Comment by mkraska** [comment]
I don't think that this approach will work, because the stress-strain curve is defined in terms of equivalent stress and equivalent strain, which both can't be negative.

**Comment by mkraska** [comment]
Theoretically, a non-symmetric stress-strain curve (yield curve) could be obtained by using kinematic hardening with non-zero initial back stress. This, however, would not affect stiffness.

**Comment by mkraska** [comment]
How about a combination of compression-only and linear elastic material? Essentially, you copy each element and assign the second material to the copies.
